examples/CMakeLists.txt

```diff
@@ -12,3 +12,6 @@ add_subdirectory(fft_physical_x)
 add_subdirectory(fft_physical_z)
 add_subdirectory(halo_test)
 add_subdirectory(io_test)
+
+# Set real/complex tests
+set(COMPLEX_TESTS "OFF" CACHE STRING "Enables complex numbers for tests that support it")
```

Comment:
Currently, the test is :

- Default : off, test with real only
- On, test with complex only

I think it is best to test both real and complex variables when the test is on

Reply:
I am having a look at this

Reply:
I would like to merge this. What is the consensus? Do we do a test only for real, real/complex(current version) or real and complex?   There are quite a bit of `#ifdef` i would like to move the test to have it only for `#ifdef ADIOS2`